static/css/techreport/techreport.css

```
@@ -224,6 +224,24 @@ nav {
   display: none;
 }
 
+/* Heading links */
+.anchor {
+  color: inherit;
+}
+
+.anchor:is(:hover, :focus-visible) {
+  position: relative;
+}
+
+.anchor:is(:hover, :focus-visible)::after {
+  content: "🔗";
```

Remove it from the accessible name (with fallback as only supported in iOS and Firefox last year - I'm OK to drop the fallback too WDYT?):

```suggestion
  content: "🔗";
  content: "🔗" / "";
```

---

I actually ended up removing the anchor symbol all together to make it consistent with the other reports, which don't have them